# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/d3d12/src/query.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/d3d12/src/query.rs b/third_party/rust/d3d12/src/query.rs
--- a/third_party/rust/d3d12/src/query.rs
+++ b/third_party/rust/d3d12/src/query.rs
@@ -1,15 +1,15 @@
-use crate::com::WeakPtr;
-use winapi::um::d3d12;
-
-#[repr(u32)]
-#[derive(Debug, Copy, Clone)]
-pub enum QueryHeapType {
-    Occlusion = d3d12::D3D12_QUERY_HEAP_TYPE_OCCLUSION,
-    Timestamp = d3d12::D3D12_QUERY_HEAP_TYPE_TIMESTAMP,
-    PipelineStatistics = d3d12::D3D12_QUERY_HEAP_TYPE_PIPELINE_STATISTICS,
-    SOStatistics = d3d12::D3D12_QUERY_HEAP_TYPE_SO_STATISTICS,
-    // VideoDecodeStatistcs = d3d12::D3D12_QUERY_HEAP_TYPE_VIDEO_DECODE_STATISTICS,
-    // CopyQueueTimestamp = d3d12::D3D12_QUERY_HEAP_TYPE_COPY_QUEUE_TIMESTAMP,
-}
-
-pub type QueryHeap = WeakPtr<d3d12::ID3D12QueryHeap>;
+use crate::com::WeakPtr;
+use winapi::um::d3d12;
+
+#[repr(u32)]
+#[derive(Debug, Copy, Clone)]
+pub enum QueryHeapType {
+    Occlusion = d3d12::D3D12_QUERY_HEAP_TYPE_OCCLUSION,
+    Timestamp = d3d12::D3D12_QUERY_HEAP_TYPE_TIMESTAMP,
+    PipelineStatistics = d3d12::D3D12_QUERY_HEAP_TYPE_PIPELINE_STATISTICS,
+    SOStatistics = d3d12::D3D12_QUERY_HEAP_TYPE_SO_STATISTICS,
+    // VideoDecodeStatistcs = d3d12::D3D12_QUERY_HEAP_TYPE_VIDEO_DECODE_STATISTICS,
+    // CopyQueueTimestamp = d3d12::D3D12_QUERY_HEAP_TYPE_COPY_QUEUE_TIMESTAMP,
+}
+
+pub type QueryHeap = WeakPtr<d3d12::ID3D12QueryHeap>;